src/views/Swap/components/AccessRisk/index.tsx

```diff
@@ -1,87 +1,208 @@
-import { useState, useEffect, useMemo } from 'react'
-import { Currency, ChainId } from '@pancakeswap/sdk'
+import { useState, useEffect, useMemo, useCallback } from 'react'
+import { Currency } from '@pancakeswap/sdk'
 import { useTranslation } from '@pancakeswap/localization'
 import { Flex, Button, HelpIcon, useTooltip, Text, Link, useToast } from '@pancakeswap/uikit'
-import { fetchRiskToken, TokenRiskPhases, RiskTokenInfo } from 'views/Swap/hooks/fetchTokenRisk'
+import { fetchRiskToken, RiskTokenInfo } from 'views/Swap/hooks/fetchTokenRisk'
 import RiskMessage from 'views/Swap/components/AccessRisk/RiskMessage'
+import { tokenListFromOfficialsUrlsAtom } from 'state/lists/hooks'
+import merge from 'lodash/merge'
+import pickBy from 'lodash/pickBy'
+import { useAtomValue } from 'jotai'
 
 interface AccessRiskProps {
-  currency: Currency
+  inputCurrency: Currency
+  outputCurrency: Currency
 }
 
-const AccessRisk: React.FC<AccessRiskProps> = ({ currency }) => {
+const AccessRisk: React.FC<AccessRiskProps> = ({ inputCurrency, outputCurrency }) => {
   const { t } = useTranslation()
   const { toastInfo } = useToast()
+  const tokenMap = useAtomValue(tokenListFromOfficialsUrlsAtom)
```

Comment:
At this moment, Scanning risk only supports BSC mainnet. I think we don't have to include tokens of other networks